doc/sphinx/language/core/inductive.rst

```diff
@@ -388,6 +388,19 @@ constructions.
    consequently :n:`forall {* @binder }, @type` and its value is equivalent
    to :n:`fun {* @binder } => @term`.
 
+   This command accepts the :attr:`program` attribute.  See :ref:`program_fixpoint`.
+   The :n:`@fixannot`
+   annotation is permitted only if :attr:`program` is present.  :n:`@fixannot` can be:
+
+      :n:`measure f R`
+        where :n:`f` is a value of type :n:`X` computed on
+        any subset of the arguments and the optional term
+        :n:`R` is a relation on :n:`X`. :n:`X` defaults to :n:`nat` and :n:`R`
+        to :n:`lt`.
+
+      :n:`wf R x`
+        which is equivalent to :n:`measure x R`.
```

Comment:
This is somehow redundant with the paragraph modified above and also wrong since `@fixannot` can be `{ struct ... }` which is accepted in basic fix definitions. Furthermore, this description is specific to the program case, so should not be here.

Reply:
OK, dropped this chunk but kept the first sentence "This command accepts the :attr:`program` attribute."